docs/Getting Started/RHEL-based distro/RHEL 8-based distro Root on ZFS/1-preparation.rst

```diff
@@ -0,0 +1,129 @@
+.. highlight:: sh
+
+Preparation
+======================
+
+.. contents:: Table of Contents
+   :local:
+
+#. Download a variant of Fedora 34 live image
```

Comment:
Fedora -> RH?

Reply:
Ah, nevermind, you use it just as live image

Reply:
For some reason, RHEL and derivatives discontinued live images starting from Version 8. Only installer is provided. Therefore Fedora is used here.